test/Tests/src/Data/Time/Duration_Spec.enso

```diff
@@ -62,8 +62,8 @@ spec =
                     (duration - period).should_fail_with Time_Error
                     (period + duration).should_fail_with Time_Error
                     (period - duration).should_fail_with Time_Error
-                    (duration > period).should_fail_with Type_Error
-                    (duration < period).should_fail_with Type_Error
+                    (duration > period).should_fail_with Incomparable_Values.Error
+                    (duration < period).should_fail_with Incomparable_Values.Error
```

Comment:
```suggestion
                    (duration > period).should_fail_with Incomparable_Values
                    (duration < period).should_fail_with Incomparable_Values
```

Since #5853, one cannot use Atom Constructors in `should_fail_with`, you need to use the _type_.

Reply:
Thx for spotting this. Seems like a mistake after merging with latest develop.